### ISSUE TITLE
Why is progressbar mentioned in F103?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#892]
https://www.w3.org/WAI/WCAG21/Techniques/failures/F103

Progressbar in F103 is listed as a live region, but it is not a live region: https://www.w3.org/TR/wai-aria-1.1/#progressbar

Even worse: a progressbar can never be turned into a live region, because the change of the value of the progressbar does not affect any output in a live region (according to the specification of live regions)

See also: https://github.com/w3c/aria-practices/issues/1165

**Comment by alastc** [comment]
I'm assuming the use of progressbar in F103 would be because you could consider a loading message as a status message, therefore the page should not fail if that has been added.

Pinging @mbgower as he created this technique.

**Comment by JAWS-test** [comment]
The ARIA specification names all roles with implicit Live Region. `progressbar` is not one of them. If a loading message is implemented only with `role=progressbar`, it does not fulfill SC 4.1.3. Therefore, the list item should be removed.

**Comment by bruce-usab** [comment]
I am not sure what the change should be, but I did want to affirm that progress bars were one of the examples of “auto-updating information” we had in mind when 2.2.2 was written.

**Comment by JAWS-test** [comment]
> I am not sure what the change should be

Just remove the list entry "role=progressbar" from the list because it is not a live region
